staff/jaume-serradell/spotify-app/scripts/main.js

```diff
@@ -0,0 +1,94 @@
+// my presentation logic
+
+// optional, reduce the size of the restaurants loaded in memory
+// restaurants.splice(100);
```

Comment:
You should clean your code! In this file os Spotify App, these two coments are not necessary; you are not working with restaurants. : D

Reply:
Done!